ds3-autogen-python/src/main/java/com/spectralogic/ds3autogen/python/PythonCodeGenerator.java

```diff
@@ -15,15 +15,124 @@
 
 package com.spectralogic.ds3autogen.python;
 
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
 import com.spectralogic.ds3autogen.api.CodeGenerator;
 import com.spectralogic.ds3autogen.api.FileUtils;
 import com.spectralogic.ds3autogen.api.models.Ds3ApiSpec;
+import com.spectralogic.ds3autogen.api.models.Ds3Request;
+import com.spectralogic.ds3autogen.api.models.Ds3Type;
+import com.spectralogic.ds3autogen.python.generators.request.BaseRequestGenerator;
+import com.spectralogic.ds3autogen.python.generators.request.RequestModelGenerator;
+import com.spectralogic.ds3autogen.python.helpers.PythonHelper;
+import com.spectralogic.ds3autogen.python.model.commands.CommandSet;
+import com.spectralogic.ds3autogen.python.model.request.BaseRequest;
+import com.spectralogic.ds3autogen.utils.Helper;
+import com.spectralogic.ds3autogen.utils.collections.GuavaCollectors;
+import freemarker.template.*;
+import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
 import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static com.spectralogic.ds3autogen.utils.ConverterUtil.isEmpty;
+import static com.spectralogic.ds3autogen.utils.ConverterUtil.removeUnusedTypes;
 
 public class PythonCodeGenerator implements CodeGenerator {
+
+    private static final org.slf4j.Logger LOG = LoggerFactory.getLogger(PythonCodeGenerator.class);
```

Comment:
I used shortcuts to fix the error. It usually imports, but i guess this time it used the qualified name. I will fix